R/analyse_IRSAR.RF.R

```diff
@@ -197,8 +197,10 @@
 #' If only one value is provided this will be treated as minimum value and the
 #' maximum limit will be added automatically.
 #'
-#' @param method [character] (*with default*): select method applied for the data analysis.
-#' Possible options are `"FIT"`, `"SLIDE"`, `"VSLIDE"`.
+#' @param method [character] (*with default*): select method applied for the
+#' data analysis. Possible options are `"FIT"`, `"SLIDE"`, `"VSLIDE"`;
+#' `"None"` can be used to disable the analysis and plot the natural points
```

Comment:
For consistency I would suggest "NONE" and internal ignore the style of writing.